index.bs

```diff
@@ -904,21 +906,30 @@ of a given agent's behavior over time.
 Delegation {#delegation}
 ----------
 
-Client Hints will be delegated from top-level pages via Permissions Policy. This reduces the likelihood that [=user agent=]
-information will be delivered along with subresource requests, which reduces the potential for
+Client Hints will be delegated from top-level pages via Permissions Policy
+([[!permissions-policy-1]]). This reduces the likelihood that [=user agent=] information will be
+delivered along with subresource requests, which reduces the potential for
 [=passive fingerprinting=].
 
 That delegation is defined as part of [=append client hints to request=].
 
 Fingerprinting {#fingerprinting}
 --------------
 
-The primary goal of User Agent Client Hints is to reduce the default entropy
-available to a server for [=passive fingerprinting=]. However, it will still be possible
-for some, or all, hints to be requested and used for [=active fingerprinting=] purposes by
-first or delegated third parties. As noted in [[#access]], [=User agents=] should consider
-policies to restrict or reduce access to parties that are known to <a lt="active fingerprinting">
-actively fingerprint</a> their users.
+The *primary goal* of User Agent Client Hints is to reduce the amount of default entropy exposed to
+the web at large through the User-Agent header field, which may be used for
+[=passive fingerprinting=] purposes.
+
+[=User agents=] naturally vary in the types and values of information exposed by the UA
+string and some have made deliberate privacy-focused choices to not expose certain bits, such
+as device [=model=] on mobile. This specification makes no requirements for a [=user agent=] to
+expose more entropy via User Agent Client Hints than it does through the User-Agent header field
+today—the empty string is a valid value for all such high-entropy hints.
```

Comment:
Is an empty string really what you are requiring when a user (or user agent) decides not to share something?

I might phrase this last sentence as:

> A [=user agent=] is able to decide which hints it provides, with the expectation that no more information is provided than would be included in an unredacted User-Agent header field.  A user agent can provide an empty string for any value that it does not wish to provide.

That piece about the empty string needs to be better captured in the specification.  I don't see this anywhere else.  The opposite in fact: the [getHighEntropyValues](https://wicg.github.io/ua-client-hints/#getHighEntropyValues) definition would seem to reject if someone asked for something that the browser (or user) didn't want to share.

Reply:
Thanks for the suggested wording, I like it.

> Is an empty string really what you are requiring when a user (or user agent) decides not to share something?

We have a couple of options: return the hint with a lie, return the hint with the empty string, or refuse to return the header at all (in the case of HTTP anyways, which is I guess functionally equivalent to not supporting the hint). Those all sound like valid choices to me, so I'll tweak the language a bit.